ci/docker/runtime_functions.sh

```diff
@@ -925,13 +925,23 @@ nightly_test_javascript() {
 nightly_model_backwards_compat_test() {
     set -ex
     export PYTHONPATH=/work/mxnet/python/
+
+    cd /work/venv
+    virtualenv -p python3
+    source /work/venv/bin/activate
```

Comment:
If we do a virtualenv why not do pip install instead of exporting PYTHONPATH?

Also, why not relative paths?

Reply:
python, python2, python3 are different now, as their corresponding pip, pip2, pip3
Maybe it would be better to make 2 different virtual env's for python2 and python3. Will try that.

Reply:
Why do we want to go for a virtual env here ?

Reply:
Nevermind. It was just an attempt to insure python and pip are in sync.